src/bin/vip.js

```diff
@@ -102,7 +102,15 @@ const rootCmd = async function() {
 			return;
 		}
 
-		Token.set( token.raw );
+		try {
+			Token.set( token.raw );
+		} catch ( e ) {
+			await trackEvent( 'login_command_token_submit_error', {
+				error: e.message,
+			} );
+
+			throw e;
```

Comment:
Should we output an error message and bail instead of throwing here?

Reply:
My thinking was that we'd fall back to the uncaught error handler which includes the pointer to contact us with the backtrace.